scripts/pkg/install.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
+
+set -e
+
+reset="\033[0m"
+red="\033[31m"
+green="\033[32m"
+yellow="\033[33m"
+cyan="\033[36m"
+white="\033[37m"
+
+printf "\n$yellow Installing Serverless!$reset\n\n"
+
+# Detect platform
+if [[ $OSTYPE == "linux-gnu" ]]; then
+  PLATFORM="linux"
+elif [[ $OSTYPE == "darwin"* ]]; then
+  PLATFORM="macos"
+else
+  echo "$red Sorry, there's no serverless binary installer available for this platform. Please open request for it at Serverless GitHub repository.$reset"
+  exit 1
+fi
+
+# Detect architecture
+MACHINE_TYPE=`uname -m`
+if [[ $MACHINE_TYPE == "x86_64" ]]; then
+  ARCH='x64'
+else
+  echo "$red Sorry, there's no serverless binary installer available for $MACHINE_TYPE architecture. Please open request for it at Serverless GitHub repository.$reset"
```

Comment:
Should we also include a link to the repository?

Reply:
Added